.travis.yml

```diff
@@ -0,0 +1,54 @@
+language: minimal
+
+branches:
+  only:
+    - master
+    - nightly-tests
```

Comment:
Should we remove the nightly-tests branch before merging?

Reply:
Yes, as all references to nightly tests too !